utils/src/utils/AzExtFsExtra.ts

```diff
@@ -81,19 +81,48 @@ export namespace AzExtFsExtra {
         return JSON.parse(file) as T;
     }
 
-    export async function writeJSON(resource: Uri | string, contents: string | unknown): Promise<void> {
+    export async function writeJSON(resource: Uri | string, contents: string | unknown, spaces?: string | number): Promise<void> {
         let stringified;
         if (typeof contents === 'string') {
             // ensure string is in JSON object format
             JSON.parse(contents);
             stringified = contents;
         } else {
-            stringified = JSON.stringify(contents);
+            stringified = JSON.stringify(contents, undefined, spaces);
         }
 
         await writeFile(resource, stringified);
     }
 
+    export async function readDirectory(resource: Uri | string): Promise<{ fsPath: string, name: string, type: FileType }[]> {
```

Comment:
These didn't get added to the index.d.ts file

Reply:
This is why I don't push code during meetings 😅